airflow/jobs/scheduler_job.py

```diff
@@ -817,16 +208,16 @@ def _change_state_for_tis_without_dagrun(
         """
         tis_changed = 0
         query = (
-            session.query(models.TaskInstance)
-            .outerjoin(models.TaskInstance.dag_run)
-            .filter(models.TaskInstance.dag_id.in_(list(self.dagbag.dag_ids)))
-            .filter(models.TaskInstance.state.in_(old_states))
+            session.query(TI)
```

Comment:
I think I'd rather we kept this as `TaskInstance` or `models.TaskInstance` instead of `TI`.


But either way _this_ type of change should be in a different PR -- this is already a big PR, so keeping changes to a minimum helps the reviewer.